nomad/jitsi_packs/packs/jitsi_meet_backend/templates/jitsi_meet_backend.nomad.tpl

```diff
@@ -557,6 +557,9 @@ EOF
         JWT_ENABLE_DOMAIN_VERIFICATION="true"
         JWT_ACCEPTED_ISSUERS="[[ env "CONFIG_jwt_accepted_issuers" ]]"
         JWT_ACCEPTED_AUDIENCES="[[ env "CONFIG_jwt_accepted_audiences" ]]"
+[[- if eq (env "CONFIG_prosody_enable_wait_for_host") "true" -]]
```

Comment:
`-]]` just means that it deletes the characters to bring the next line up to wherever this line starts. it might actually do the wrong thing because you have it at both ends. it is sometimes hard to tell until the template renders.

Reply:
```
        JWT_ACCEPTED_AUDIENCES="[[ env "CONFIG_jwt_accepted_audiences" ]]"
[[- if eq (env "CONFIG_prosody_enable_wait_for_host") "true" -]]
        JWT_ACCEPTED_ALLOWNER_ISSUERS="{ \"jitsi\" }"
[[- end -]]
```

will probably result in
```
        JWT_ACCEPTED_AUDIENCES="FOO"JWT_ACCEPTED_ALLOWNER_ISSUERS="{ "jitsi" }"
```

you probably want
```
        JWT_ACCEPTED_AUDIENCES="[[ env "CONFIG_jwt_accepted_audiences" ]]"
[[- if eq (env "CONFIG_prosody_enable_wait_for_host") "true" ]]
        JWT_ACCEPTED_ALLOWNER_ISSUERS="{ \"jitsi\" }"
[[- end ]]
```